adapters/tappx/tappx_test.go

```diff
@@ -38,13 +38,16 @@ func TestTsValue(t *testing.T) {
 
 	bidderTappx := bidder.(*TappxAdapter)
 
-	var test int
-	test = 0
+	var test int = 0
```

Comment:
Surprised there's not a lint error for setting this to it's default value. var test int` or `test := 0` would be better.

Reply:
done (changed to `test := 0`)